metpy/plots/tests/test_declarative.py

```diff
@@ -150,6 +152,29 @@ def test_projection_object():
     return pc.figure
 
 
+def test_colorfill():
+    """Test that we can use ContourFillPlot."""
+    data = xr.open_dataset(get_test_data('narr_example.nc', as_file_obj=False))
+
+    contour = ContourFillPlot()
+    contour.data = data
+    contour.level = 700 * units.hPa
+    contour.field = 'Temperature'
+    contour.colormap = 'coolwarm'
+    contour.colorbar = 'vertical'
+
+    panel = MapPanel()
+    panel.area = (-110, -60, 25, 55)
+    panel.layers = [cfeature.STATES]
+    panel.plots = [contour]
+
+    pc = PanelContainer()
+    pc.panel = panel
+    pc.draw()
+
+    return pc.figure
```

Comment:
If you're returning `pc.figure`, that implies this should be an image comparison test, and you'll need a `@pytest.mark.mpl_image_compare` above the test name, as well as commit a baseline image. Probably worth doing for this "new" plot type.

Reply:
Done.